librz/bin/format/dmp/dmp64.c

```diff
@@ -46,6 +46,11 @@ static bool rz_bin_dmp64_init_triage(struct rz_bin_dmp64_obj_t *obj) {
 	return true;
 }
 
+static void dmp64_endienness_memory_run(dmp_p_memory_run *run) {
```

Comment:
`endienness` sounds weird to me. I would call these functions something like `dmp64_memory_run_endian_to_host` similar to e.g. `ntohl`/`htonl`.